packages/web3-eth/types/tests/eth.tests.ts

```diff
@@ -18,13 +18,22 @@
  * @date 2018
  */
 
+import {Log} from 'web3-core';
 import {
```

Comment:
why are you importing them when its not being tested?

Reply:
I have imported them as a test. This should give us the proof those forwarded types do exist in the ``web3-eth`` package. :)